openmicroscopy/openmicroscopy
# filter rating

Simple filter by rating functionality for webclient centre panel (as requested by Ilan).

To test,
- Rate images 1-5. Also log in as another user and rate some images.
- Add Filters using chooser (rating or name)
- Rating - click on stars.
- Rating matches exact number, any user (E.g. 3 stars matches any image that has been rated 3 by any user)
- Filter by text works as before (filters for any text on each row in table).
- X is shown on hover and clicking this will hide and clear filter
- Try adding and clearing filters in various combinations
- If a range of images are selected before filtering - filtered images will be de-selected.

<img width="482" alt="screen shot 2016-06-08 at 23 33 38" src="https://cloud.githubusercontent.com/assets/900055/15913381/8bb3bd9e-2dd1-11e6-9d19-a2ef27228892.png">
